List of AI bots Cloudflare considers "Verified" (#79)
ai-robots-txt/ai.robots.txt: README.md

If you use [Cloudflare's hard block](https://blog.cloudflare.com/declaring-your-aindependence-block-ai-bots-scrapers-and-crawlers-with-a-single-click) alongside this list, you can report abusive crawlers that don't respect `robots.txt` [here](https://docs.google.com/forms/d/e/1FAIpQLScbUZ2vlNSdcsb8LyTeSF7uLzQI96s0BKGoJ6wQ6ocUFNOKEg/viewform).

> 
> ## Cloudflare Verified Bots
> 
> If you are unable to make use of [Cloudflare's hard block](https://blog.cloudflare.com/declaring-your-aindependence-block-ai-bots-scrapers-and-crawlers-with-a-single-click) and/or have WAF rules that use the `cf.bot_management.verified_bot` condition based on [Cloudflare's Verified Bots](https://radar.cloudflare.com/traffic/verified-bots), please note that the following AI web crawlers are considered verified bots by Cloudflare:
> - Amazonbot
> - Applebot
> - CCBot
> - ChatGPT-User
> - DuckAssistBot
> - GoogleOther
> - GPTBot
> - OAI-SearchBot
> - PerplexityBot
> - PetalBot
> 

Reviewer: I'd suggest simplifying this PR and avoiding the maintenance overhead (think a continual succession of future PRs) of duplicating their list of verified bots. Also, I don't think we need to speculate about how the list might be used. That's my personal perspective anyway.

```suggestion
But even if you don't use Cloudflare's hard block, their list of [verified bots](https://radar.cloudflare.com/traffic/verified-bots) may come in handy.
```

Author: Thank you @glyn, I agree this is a much better way to go about it.
